packages/gatsby/src/redux/__tests__/__snapshots__/pages.js.snap

```diff
@@ -114,6 +120,7 @@ Array [
     "path": "/hi/",
     "pluginCreatorId": "test",
     "pluginCreator___NODE": "test",
+    "updatedAt": "2017-08-23T02:25:12.557Z",
   },
 ]
 `;
```

Comment:
Ugh, those were from a PR of mine and I guess I didn't see Travis failing. Sorry about that.